src/test/java/org/trd/app/teknichrono/model/manage/TestLapTimeOrder.java

```diff
@@ -292,13 +293,13 @@ public void fillGapsInfoWhenOrderedByDuration() {
       if (previousGap == -1) {
         Assert.assertEquals(0, l.getGapWithBest());
       }
-      if (l.getDuration() > 0) {
+      if (l.getDuration().compareTo(Duration.ZERO) > 0) {
         Assert.assertTrue(l.getGapWithBest() >= previousGap);
       } else {
         // No more gaps
-        previousGap = Long.MAX_VALUE;
+        previousGap = Long.MAX_VALUE; // FIXME this is a bug, always overridden
```

Comment:
Can you explain the FIXME ?

Reply:
write value was always overridden two lines later